stl/inc/xutility

```diff
@@ -7235,6 +7239,9 @@ _NODISCARD constexpr bool _Mul_overflow(const _Int _Left, const _Int _Right, _In
 }
 #endif // _HAS_CXX23 && defined(__cpp_lib_concepts)
 
+// TRANSITION, non-_Ugly attribute tokens
+#pragma pop_macro("msvc")
+
 _STD_END

```

Comment:
It doesn't really matter, but I think we should be consistent here and pretend to respect scope. The `push_macro` on 29 occurs before the `_STD_BEGIN` far below on ~199.